podcasts/Onboarding/Plus/Account Prompt/PlusAccountPromptTableCell.swift

```diff
@@ -12,9 +12,17 @@ class PlusAccountPromptTableCell: ThemeableCell {
     override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
         super.init(style: style, reuseIdentifier: reuseIdentifier)
 
-        let view = PlusAccountUpgradePrompt(viewModel: model, contentSizeUpdated: { [weak self] size in
-            self?.contentSizeUpdated?(size)
-        }).themedUIView
+        let view: UIView
+        if FeatureFlag.newAccountUpgradePromptFlow.enabled {
+            let _ = OnboardingFlow.shared.begin(flow: .plusAccountUpgrade, in: model.parentController, source: PlusAccountPromptViewModel.Source.profile.rawValue, context: nil)
```

Comment:
Just calling this method to get the flow and source correctly set for the tracking events, but then I'm ignoreing the VC because we don't need it here.